src/main/target/CC3D/target.h

```diff
@@ -72,8 +72,13 @@
 #define USE_VCP
 #define USE_UART1
 #define USE_UART3
+
+#ifdef CC3D_OPBL
```

Comment:
No. I like to keep the serial port config issue separate, as there is an else branch here. Not so in the other items. Well, they could be.... Makes the code somewhat more readable. 
Why would one #ifdef be more important?